src/Tests/AuthoringTest/Program.cs

```diff
@@ -207,6 +207,13 @@ public sealed partial class CustomProperty
         public string Value => "CsWinRT";
     }
 
+    [GeneratedBindableCustomProperty]
+    public sealed partial struct CustomPropertyStructType

```

Comment:
Should we add a private / internal record which we return as a boxed value which we then do ICustomProperty on?  Or at least a compile time record test case.

Reply:
![WhyThoGIF](https://github.com/user-attachments/assets/8035a352-d25f-481c-af8c-51c7f2b56ceb)